src/magnet_uri.cpp

```diff
@@ -190,7 +190,8 @@ namespace libtorrent {
 				error_code e;
 				display_name = unescape_string(value, e);
 			}
-			else if (name == "tr"_sv) // tracker
+			else if (name == "tr"_sv ||
+			    (name.substr(0, 3) == "tr."_sv && is_numeric(name.substr(3)))) // tracker
```

Comment:
Since we don't care or use the value following the dot should we even check if it's numeric?